src/README.md

```diff
@@ -0,0 +1,67 @@
+# Variant Nowcast Hub Scripts
+
+The `src/` directory contains scripts that support this hub's operations. The scripts are designed to be run by scheduled
+GitHub workflows on a Linux-based runner (_i.e._, they have not been tested in a Windows environment).
+
+The sections below contain information about the scripts and how to run them manually.
+
+## Generating list of clades to model
+
+`get_clades_to_model.py` generates a list of clades to model for the hub's upcoming round (the first Wednesday following
+the run date). The script writes the clade list and accompanying metadata to `auxiliary-data/modeled-clades/[round_id].txt`.
+
+To run the script manually:
+
+1. Make sure that `uv` is installed on your machine:
+
+    ```bash
+    brew install uv
+    ```
+
+    (see [`uv` documentation](https://docs.astral.sh/uv/getting-started/installation/#installing-uv) for a full list of installation options)
+
+2. From the root of the repo, run the following command:
+
+    ```bash
+    uv run src/get_clades_to_model.py
```

Comment:
I got the following error 

```
$ uv run src/get_clades_to_model.py 
Reading inline script metadata from: src/get_clades_to_model.py
 Updated https://github.com/reichlab/virus-clade-utils/ (d3a5874)
  × No solution found when resolving script dependencies:
  ╰─▶ Because the current Python version (3.10.14) does not satisfy Python>=3.11 and
      virus-clade-utils==0.1.0 depends on Python>=3.11, we can conclude that virus-clade-utils==0.1.0
      cannot be used.
      And because only virus-clade-utils==0.1.0 is available and you require virus-clade-utils, we can
      conclude that your requirements are unsatisfiable.
```

I have my python via pyenv and was able to get this to work by setting `pyenv local 3.11`, which wrote a `.python-version` file. 


After that, it worked:

```
2024-09-23T14:08:58.307133Z [info     ] genome metadata downloaded     elapsed=49.09401858299998 filename=sequence.py lineno=84
2024-09-23T14:09:18.611833Z [info     ] generated clade list           elapsed=20.294295000000147 filename=get_clade_list.py lineno=64
09/23/2024 07:09:18 AM -  INFO - Clade list: ['24A', '24B', '24C', 'recombinant']
09/23/2024 07:09:18 AM -  INFO - Clade list saved: /path/to/variant-nowcast-hub/auxiliary-data/modeled-clades/2024-09-25.txt
```

Reply:
Good spot. The great thing about running the script via `uv` is that you don't need to worry about having python installed at all (no futzing! https://docs.astral.sh/uv/guides/scripts/)

The problem here is that I assumed `uv run` would detect the `.python-version` file in `src`, even when executing it from the hub's root directory. My laptop and the GitHub actions runner both use Python 3.12, so I didn't catch this.

Submitted a patch here: https://github.com/reichlab/variant-nowcast-hub/pull/53